README.md

```diff
@@ -17,8 +16,7 @@ The idea is to help doctors monitor large numbers of patient's symptoms over tim
 
 We will scope it out to creating a dashboard and a survey that people update daily with information.
 
-In a few words or less, this will *"track and reduce symptoms of COVID-19."*
-
+In a few words or less, this will *"track symptoms of COVID-19 to reduce doctor overhead."*
```

Comment:
I made just a bit of a change to our few-words-or-less goal, let me know if anyone has any suggestions here. I personally felt that to be true to what we're building, we ourselves won't effectively reduce any symptoms (as we aren't medical professionals and don't want to come across as experts as was stressed in that call). But, we will allow doctors to more effectively treat an ever-growing number of cases and people thinking they are showing symptoms / might be sick.

Reply:
Good points. I like the edits.